util/conversion/conversion.go

```diff
@@ -139,10 +139,10 @@ func GetFuzzer(scheme *runtime.Scheme, funcs ...fuzzer.FuzzerFuncs) *fuzz.Fuzzer
 type FuzzTestFuncInput struct {
 	Scheme *runtime.Scheme
 
-	Hub conversion.Hub
+	Hub              conversion.Hub
 	HubAfterMutation func(conversion.Hub)
 
-	Spoke conversion.Convertible
+	Spoke              conversion.Convertible
```

Comment:
Where are these changes coming from?

Reply:
I ran go fmt..... on the code this isn't necessary I'll remove